exchanges/orderbook/orderbook_types.go

```diff
@@ -65,6 +74,8 @@ type Base struct {
 	LastUpdateID int64         `json:"lastUpdateId"`
 	AssetType    asset.Item    `json:"assetType"`
 	ExchangeName string        `json:"exchangeName"`
+	// NotAggregated defines expected duplication of prices from endpoint
```

Comment:
More a nit of my own dumbness, could this be changed to `// NotAggregated defines whether an orderbook can contain duplicate prices in a payload`